woocommerce/woocommerce-gateway-stripe
# chore: add JS/CSS lint & workflow

# Changes proposed in this Pull Request:

We're not lining the JS files, so sometimes review time is spent providing suggestions on JS code styles.
Introducing the eslint workflow, so that all PRs are opened with styles in mind.

Most of the changes have been performed automagically, so when this PR gets merged, all the new changes will require to adhere to the rules.

# Testing instructions
- Make changes to a file
- Try to commit them
- Rules should kick in
